Assets/MixedRealityToolkit/_Core/Utilities/MathUtils.cs

```diff
@@ -419,5 +419,25 @@ public static Vector3 NearestPointToLinesLeastSquares(IEnumerable<Ray> rays)
             Vector3 nearestPoint = sumOfProductInverse * sumOfProductTimesDirection;
             return nearestPoint;
         }
+
+        /// <summary>
+        /// Convert degrees to radians.
+        /// </summary>
+        /// <param name="degrees">Angle, in degrees.</param>
+        /// <returns>Angle, in radians.</returns>
+        public static float DegreesToRadians(double degrees)
+        {
+            return (float)(degrees * Mathf.Deg2Rad);
+        }
+
+        /// <summary>
+        /// Convert radians to degrees.
+        /// </summary>
+        /// <param name="radians">Angle, in radians.</param>
+        /// <returns>Angle, in degrees.</returns>
+        public static double RadiansToDegrees(float radians)
+        {
+            return (double)(radians * Mathf.Rad2Deg);
```

Comment:
Does casting to double give you better precision here even though the multiplied values are float types?

Reply:
doubtful. i can make it return a float

Reply:
Well the method above it is the same. So we can probably just remove the duplicate